terra-jupyter-gatk/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+## 0.0.1 - 09/24/2019
+
+- added all from terra-jupyter-python:0.0.1
```

Comment:
can you add a readme file similar to https://github.com/DataBiosphere/terra-docker/tree/master/terra-jupyter-base